client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -1,13 +1,98 @@
-import { Card } from '@automattic/components';
-import { useTranslate } from 'i18n-calypso';
-import { ReactElement } from 'react';
+import {
+	ReloadSetupIntentId,
+	StripeHookProvider,
+	StripeSetupIntentIdProvider,
+	useStripe,
+	useStripeSetupIntentId,
+} from '@automattic/calypso-stripe';
+import { Card, Button } from '@automattic/components';
+import { CheckoutProvider, CheckoutSubmitButton } from '@automattic/composite-checkout';
+import { isValueTruthy } from '@automattic/wpcom-checkout';
+import { CardElement, useElements } from '@stripe/react-stripe-js';
+import { useSelect } from '@wordpress/data';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import page from 'page';
+import { useCallback, useMemo, ReactElement, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import CardHeading from 'calypso/components/card-heading';
 import DocumentHead from 'calypso/components/data/document-head';
 import Main from 'calypso/components/main';
+import CreditCardLoading from 'calypso/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-loading';
+import PaymentMethodImage from 'calypso/jetpack-cloud/sections/partner-portal/credit-card-fields/payment-method-image';
+import { assignNewCardProcessor } from 'calypso/jetpack-cloud/sections/partner-portal/payment-methods/assignment-processor-functions';
+import { getStripeConfiguration } from 'calypso/jetpack-cloud/sections/partner-portal/payment-methods/get-stripe-configuration';
+import { useCreateStoredCreditCardMethod } from 'calypso/jetpack-cloud/sections/partner-portal/payment-methods/hooks/use-create-stored-credit-card';
 import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { getCurrentUserLocale } from 'calypso/state/current-user/selectors';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
```

Comment:
I don't have an opinion on this particularly but FWIW we used to have this style of organization in calypso too but they've been removed since calypso now has a eslint plugin which auto-organizes the imports in a file.

Reply:
>calypso now has a eslint plugin which auto-organizes the imports in a file

Aaah! I didn't know that. 

> Calyspo example: client/my-sites/site-settings/settings-p2/preapproved-domains.jsx

I wonder why the plugin hasn't done anything on `client/my-sites/site-settings/settings-p2/preapproved-domains.jsx` since it's relatively new and has had recent changes. But that's outside the scope of this PR 😄